ci/pipeline

```diff
@@ -263,14 +263,28 @@ def release(args: String*): Unit = {
 
   %('git, "checkout", config.version.commit)
 
-  buildDockerAndLinuxPackages()
-  if (config.runTests)
-    testDockerAndLinuxPackages()
+  val targets = config.targets.toSet
+
+  if (targets.contains(ReleaseTarget.LinuxPackages) ||
```

Comment:
I think this will break the dry run feature? The intention with the dry run feature is that we would have a way to run the compile and test phases without producing any external side effects.

May be, instead, we could add a dependencies field to each of the release targets, and add a release target of "Compile", "Test", and, "PackageTest", with respective dependencies.

Then, below, instead of "targets.foreach", we could invoke something like:

```
targets.toSeq.flatMap { t => t.recursiveDependencies :+ t }.distinct.foreach { t =>
  ...
}
```

(We could not use a set as those are unordered)

This would cause the graph execution order to be worked out, naturally, without too much code.

Reply:
Good point.